packages/conf-rust-llvm/conf-rust-llvm.1/opam

```diff
@@ -17,7 +17,7 @@ depexts: [
   ["rust-llvm"] { os-family = "alpine" }
   ["rust-llvm"] { os-family = "arch" }
   ["rust-llvm"] { os-family = "debian" }
-  ["rust-llvm"] { os-family = "ubuntu" }
+  [] { os-family = "ubuntu" }
```

Comment:
```suggestion
  # ["rust-llvm"] { os-family = "ubuntu" } # rust-lld is provided by the rustc package installed by conf-rust which depends on cargo
```

Reply:
To avoid doing anything the depext should go altogether. Not sure if that was the culprit but it is better to remove the line or comment it out  with an explanation. I was looking for the moment rust-llvm was removed from Ubuntu to eventually add it just for the versions that needed it